### ISSUE TITLE
Unable to exclude fonts in Windows 10

### ISSUE THREAD

**Issue by tordenflesk** [snowie2000/mactype#626]
Proof:
https://i.imgur.com/byHacPt.png
CMDER is able to disable ClearType. The same ini is able to exclude "pixel" fonts in Windows 7:
; A technically and visually correct gamma for sRGB or Rec709 displays
; By mufunyo
; Version 1.0

[Preview]
Font=Helvetica
Color=$000000
Text=A technically and visually correct gamma for sRGB or Rec709 displays by extratype 01234567890!@#$%^&*
Size=10

[General]
Name=mufunyo-flesk
HookChildProcesses=1
HintingMode=1
AntiAliasMode=4
NormalWeight=0
BoldWeight=0
ItalicSlant=-2
UseMapping=0
GammaMode=0
GammaValue=1.4
Contrast=1.0
RenderWeight=1.0
TextTuning=0
TextTuningR=0
TextTuningG=0
TextTuningB=0
BolderMode=2
FontLoader=0
Fontlink=2
FontSubstitutes=1
MaxHeight=0
LcdFilter=2
LoadOnDemand=1
CacheMaxFaces=256
CacheMaxSizes=33554432
CacheMaxBytes=67108864
EnableKerning=1
MaxBitmap=0
HintSmallFont=1
DirectWrite=1
Shadow=1,1,0,0x0,0,0x0

[Experimental]
ColorFont=1

[Experimental@chrome.exe]
; Make color fonts appear correctly in Chrome
InvertColor=1

[Experimental@idea64.exe]
; Workaround for IDEA/JAVA font rendering unverified as of 2018/10/19
clipboxfix=1

[ExcludeModule]
[exclude]
cure
drift
gelly
lime
marlett
MonteCarlo Fixed 12
NU
PF Tempesta Seven
PF Tempesta Seven Condensed
scientifica
scientificaCAPS
Semplice Extended
Semplice Regular
Semplice Regular Regular
snap
[FontSubstitutes]
[Individual]
Courier New=,,22,,,

[DirectWrite]
GammaValue=1.8
Contrast=0
ClearTypeLevel=1.0
RenderingMode=5

[ExcludeSub]
adb.exe
bbLeanSkinRun32.exe
blackbox.exe
ConEmu64.exe
ConEmuC64.exe
Content Manager.exe
dottorrent-gui.exe
dupeGuru.exe
dupeguru-win64.exe
getsky.exe
kitty.exe
makemkv.exe
makemkvcon64.exe
partitionwizard.exe
picard.exe
qbittorrent.exe
Steam.exe
steamwebhelper.exe
vlc.exe
vmware.exe
vsp.exe
Wox.exe
[UnloadDll]
adb.exe
AdobeIPCBroker.exe
audiodg.exe
bbLeanSkinRun32.exe
blackbox.exe
CCXProcess.exe
ConEmu64.exe
ConEmuC64.exe
dottorrent-gui.exe
dupeGuru.exe
dupeguru-win64.exe
fontforge.exe
getsky.exe
igfxCUIService.exe
igfxEM.exe
igfxHK.exe
kitty.exe
lass.exe
makemkv.exe
makemkvcon64.exe
MsMpEng.exe
node.exe
NVDisplay.Container.exe
OfficeClickToRun.exe
partitionwizard.exe
pia-service.exe
picard.exe
putty.exe
qbittorrent.exe
QHActiveDefense.exe
QHSafeTray.exe
QHWatchdog.exe
RuntimeBroker.exe
SearchFilterHost.exe
SearchIndexer.exe
SearchProtocolHost.exe
services.exe
spoolsv.exe
Spotify.exe
stacsv64.exe
Steam.exe
steamwebhelper.exe
SynTPEnh.exe
SynTPHelper.exe
taskhostw.exe
ts3client_win64.exe
VBoxSVC.exe
VirtualBox.exe
vlc.exe
vmnat.exe
vmnetdhcp.exe
vmware.exe
vmware-authd.exe
vmware-usbarbitrator64.exe
vmware-vmx.exe
Wox.exe

**Comment by snowie2000** [comment]
Please upload your profile directly as sometimes the charset matters.

**Comment by tordenflesk** [comment]
[mufunyo-flesk.txt](https://github.com/snowie2000/mactype/files/3995183/mufunyo-flesk.txt)

**Comment by snowie2000** [comment]
I didn't quite get your idea.
MacType excludes all bitmap fonts by default as freetype can do no better than ClearType. However, there is a bug introduced by Windows 10 which prevents MacType from identifying those fonts. I guess this is what caused a lot of crashes in Windows 10 recently, but not the issue in your case as you already excluded all the bitmap fonts manually.

Could you please post snapshots of the correct and incorrect rendering result in your case?

**Comment by tordenflesk** [comment]
Font: Scientifica (scientifica.ttf), W10 ("Smooth edges of screen fonts" ON):
https://i.imgur.com/w2DC0uS.png
Font: Scientifica (scientifica.ttf), W10 ("Smooth edges of screen fonts" OFF):
https://i.imgur.com/q9UETCP.png
Font: Semplice Regular (SEMPRG_.TTF), W7 ("Smooth edges of screen fonts" ON):
https://i.imgur.com/w5KbSg2.png

**Comment by snowie2000** [comment]
So, does it has anything to do with mactype?
MacType doesn't work on pixel font at all ...

**Comment by tordenflesk** [comment]
It most certainly does, at least on W7.
If I don't exclude them on Windows 7 they get that ugly blue-yellow effect.

**Comment by snowie2000** [comment]
There is no surprise for you to get those colorful stripes if you're indeed using the "TrueType" version of the Scientifica as it is not a pixel font in the first place.

True bitmap fonts generally end with ".fon" extension instead of ".ttf" which stands for TrueType.

You should really try everything again w/o mactype and see if it is the difference between the text rendering system across the two operating systems.

**Comment by tordenflesk** [comment]
Yes,  obviously disabling Mactype & Cleartype would not cause this issue to appear...

Being unable to exclude fonts in W10 would, logically, appear to be an issue with MacType. Which is why I made this ticket... to report the issue...

**Comment by snowie2000** [comment]
Of course, you can solve the problem by disabling mactype & cleartype, because you've completely shut down the subpixel rendering.

It's not a problem of mactype font exclusion. It's that you're using the wrong font -  a font that supports subpixel rendering even with ClearType! MacType excluded it, but cleartype is still rendering it in subpixel mode.
